packages/ember-metal/lib/properties.js

```diff
@@ -38,6 +41,16 @@ export function DEFAULT_GETTER_FUNCTION(name) {
   };
 }
 
+export function INHERITING_GETTER_FUNCTION(name) {
+  function IGETTER_FUNCTION() {
+    var proto = Object.getPrototypeOf(this);
```

Comment:
I don't think so, this exists to simulate the behavior if no descriptor existed at this level. It aims to re-target get one step up the prototype chain.

Related tests: https://github.com/emberjs/ember.js/pull/12491/files#diff-25946da80206a01a2f680ad806bb1780R370